REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.7
+OrderedCollections 1.0.1
```

Comment:
Can we just get rid of this file now?

Reply:
Nope, I checked this morning.
Attobot won't talk to you if you don't have a REQUIRE file.